app/fabric.properties

```diff
@@ -0,0 +1,3 @@
+#Contains API Secret used to validate your application. Commit to internal source control; avoid making secret public.
```

Comment:
Good point! Actually we should move all the keys away from project as recommended here: https://docs.fabric.io/android/fabric/settings/working-in-teams.html. As for now I will keep this task for the future (https://trello.com/c/dYeIJQ1i/113-remove-sensitive-data-from-github).